controllers/controller.js

```diff
@@ -9,3 +9,13 @@ exports.getTopics = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getEndpoints = (req, res, next) => {
+  fetchEndpoints()
+    .then((endpoints) => {
+      res.status(200).send({ endpoints });
+    })
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
You can also require the endpoints.json file in the controller and send this in your response. Then there's no need for a model 🙂